Assets/Content/Entities/Humanoids/Human/Human.prefab

```diff
@@ -573,6 +552,35 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 8d4b90a59a774591b9e883c310c3935a, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!54 &9076341938986001650
+Rigidbody:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 289348138277843802}
+  serializedVersion: 2
+  m_Mass: 80
+  m_Drag: 80
+  m_AngularDrag: 80
+  m_UseGravity: 1
+  m_IsKinematic: 0
+  m_Interpolate: 0
+  m_Constraints: 112
+  m_CollisionDetection: 0
+--- !u!65 &4372809320507257544
+BoxCollider:
```

Comment:
Can a CapsuleCollider feel better/more natural? I imagine that makes it harder to get caught on edges